### DISCUSSION TITLE
Copilot is not compatible with VS Code 1.46.1 on Mac OS 13.2

### DISCUSSION THREAD

**Discussion by lamaks** [community/community#46799] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

Unable to install extension on VS Code 1.46.1 on Mac OS 13.2 for both main and nightly builds. Getting the following error:

`Unable to install 'github.copilot' extension because it is not compatible with the current version of VS Code (version 1.46.1).`

Any workaround?

**Comment by davecheney** [comment, marked as answer]
I'm sorry Copilot does not support older versions of VSCode.
